submissions/5mountains/popup/index.html

```diff
@@ -0,0 +1,142 @@
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#" alt="news icon">
+                            <img src="./img/submenu/news.png">
+                            News
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#" alt="map icon">
+                            <img src="./img/submenu/maps.png">
+                            Map
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#" alt="files icon">
+                            <img src="./img/submenu/files.png">
+                            Files
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#" alt="contacts icon">
+                            <img src="./img/submenu/contacts.png">
+                            Contacts
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/translation.png" alt="translation icon">
+                            Translate
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/videos.png" alt="videos icon">
+                            Videos
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/search.png" alt="search icon">
+                            Search
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/photos.png" alt="photos icon">
+                            Photos
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/apps.png" alt="apps icon">
+                            Apps
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/documents.png" alt="documents icon">
+                            Documents
+                        </a>
+                    </li>
+                    <input type="checkbox" id="submenu__input" class="submenu__item-input">
```

Comment:
Please, add a closing mark `/` to the `input` tag.

Reply:
Is it really necessary for HTML?

Reply:
For some tags it's optional anyway I agree with @MaxKalashnyk, with the closing mark `HTML` looks more elegant and in 1 style 
Do you have any thoughts about this?

Reply:
Ok, I will! But honestly, I thought that closed input tags are not mandatory on html5, by the way, these tags have been created by emmet... hm... maybe I should set up Emmet)

Reply:
I read about two syntaxes for this element:
`
HTML - <input> `
`XHTML - <input/>
`